windows/src/developer/inst/copydev.in

```diff
@@ -81,16 +83,56 @@ heat-model-compiler:
     # operating systems, node.js is a dependency and the compiler can be installed with
     # `npm install @keymanapp/lexical-model-compiler`.
 
+    # First, we copy the files to an intermediate temp-folder rooted within repo space.
+    -rmdir /s/q $(KEYMAN_INTERMEDIATE_MODELCOMPILER)
+    mkdir $(KEYMAN_INTERMEDIATE_MODELCOMPILER)
+
+    # Performs the copy, but first copies over a file filter that is subsequently removed after the copy.
+    copy /Y $(ROOT)\src\developer\inst\exclude-node_modules $(KEYMAN_MODELCOMPILER_ROOT)
+    xcopy $(KEYMAN_MODELCOMPILER_ROOT)\* $(KEYMAN_INTERMEDIATE_MODELCOMPILER)\ /s /EXCLUDE:exclude-node_modules
+    del $(KEYMAN_MODELCOMPILER_ROOT)\exclude-node_modules
+
+    # Remove files we don't want to harvest; build a product node_modules folder
+    -rmdir /s/q $(KEYMAN_INTERMEDIATE_MODELCOMPILER)\node_modules
+
+    cd $(KEYMAN_ROOT)
+!ifdef GIT_BASH_FOR_KEYMAN
+    # attach to lerna management
+    $(GIT_BASH_FOR_KEYMAN) resources\lerna-utils.sh add-package "windows/bin/intermediate/ModelCompiler"
+    $(GIT_BASH_FOR_KEYMAN) resources\lerna-utils.sh remove-package "developer/js"
+
+    # leverage lerna to retrieve only the resources needed for production.
+    # Avoids retrieving local resources from remote (which `npm install` would do)
+    npm run lerna -- bootstrap -- --production
+
+    # detach from lerna
+    $(GIT_BASH_FOR_KEYMAN) resources\lerna-utils.sh remove-package "windows/bin/intermediate/ModelCompiler"
+    $(GIT_BASH_FOR_KEYMAN) resources\lerna-utils.sh add-package "developer/js"
+!else
+    # attach to lerna management
+    start /wait .\resources\lerna-utils.sh add-package "windows/bin/intermediate/ModelCompiler"
+    start /wait .\resources\lerna-utils.sh remove-package "developer/js"
+
+    # leverage lerna to retrieve only the resources needed for production.
+    # We're at the repo's base - ensure that Lerna is available first.
+    npm install
+    # Installs needed dependencies in repo packages (using `npm link` where possible)
+    npm run lerna -- bootstrap -- --production
+
+    # detach from lerna
+    start /wait .\resources\lerna-utils.sh remove-package "windows/bin/intermediate/ModelCompiler"
+    start /wait .\resources\lerna-utils.sh add-package "developer/js"
+!endif
+
+    # node_modules harvest complete - we can now do the rest in true $(TEMP) space.
```

Comment:
Can we do all the work in the intermediate folder and not use temp at all now?

Reply:
We could probably move some of the `rmdir` commands there, to be sure... but I'm fairly certain the 'wix' setup is dependent on the final file path, so we'd still want a final `xcopy` to the original location.  That's what my interpretation of [these lines](https://github.com/keymanapp/keyman/blob/master/windows/src/developer/inst/copydev.in#L13-L18) tells me, anyway.

Of course, I'm also less comfortable with makefiles, let alone the implications of the `.in` filetype (I see that the `.in` gets compiled into a `.mak`?) and the complicated makefile hierarchy of the Developer project as a whole, so I figured I'd leave as much of the original process in place as I could - less chances that I might break something.